### DISCUSSION TITLE
Mapsforge map writer

### DISCUSSION THREAD

**Discussion by AFA141** [mapsforge/mapsforge#1400] [Q&A] [answered]
Hello,

I am making new converter application using Osmosis library and Mapsforge map writer.
I want to make progress bar or show to user how much time this process will take?
Huge files takes a lot time to convert, so I would like to inform the user approximately how long the process will take...
Is there any way to do it?

**Comment by devemux86** [comment, marked as answer]
Map writer is a command line tool, there is a [progress-logs](https://github.com/mapsforge/mapsforge/blob/master/docs/Getting-Started-Map-Writer.md#basic-options) option.

Command line also informs about the steps of the map creation.
Better check the code to see how the write steps are processed.

**Reply by AFA141** [reply]
Hello, I know about progress-logs, problem is I am making GUI application and I want that progress logs to display into custom Text Area.
